spec/resources/workbooks_spec.rb

```diff
@@ -1,14 +1,12 @@
 require 'spec_helper'
 
+# NOTE: if it exists, delete test/testpublish workbook before recreating cassettes
```

Comment:
I thought it made sense here because you recreate each cassette separately, and I don't have a comprehensive list for all cassettes.